config/configload/backend.go

```diff
@@ -216,23 +255,20 @@ func wrapTokenRequestBackend(helper *helper, parent hcl.Body) (hcl.Body, error)
 			return nil, berr
 		}
 
+		// TODO what is this for? can it be removed?
```

Comment:
May be, the following `hclbody.MergeBodies()` would throw a panic

Reply:
`PrepareBackend()` only returns a `nil` tokenRequestBackend, if an error is returned, which is already handled by the preceding
```hcl
		if berr != nil {
			return nil, berr
		}
```